pyproject.toml

```diff
@@ -62,6 +62,7 @@ skips = ["B101"]  # https://bandit.readthedocs.io/en/latest/plugins/b101_assert_
 omit = [
     "app/logging/*",
     "app/main.py",
+    "tests/app/test_main.py"
```

Comment:
What does adding this here do? I thought our coverage tool automatically ignored tests.

Reply:
We run coverage on tests to make sure the devs' tests work like they expected. Can catch cases where a dev thought the test code was being ran but it wasn't.